packages/itwinui-css/backstop/tests/table.html

```diff
@@ -15,14 +15,27 @@
     />
     <meta charset="utf-8" />
     <title>Table | iTwinUI</title>
-    <link
-      rel="stylesheet"
-      href="../../css/all.css"
-    />
-    <link
-      rel="stylesheet"
-      href="../assets/demo.css"
-    />
+
+    <script type="module">
+      import "./assets/theme.js";
+      import "./assets/userIcon.js";
```

Comment:
Ran `yarn dev` and opened Table html. I see that user icon is not shown properly as it doesn't have its css.
![image](https://user-images.githubusercontent.com/36186912/179758607-64b73e39-7b01-4000-95e7-20837f4fc4c5.png)

Reply:
i think it might need build first, can you try changing turbo.json like this and just run `yarn dev` after that?

![dependsOn build](https://user-images.githubusercontent.com/9084735/179763083-6902a024-3371-42e7-bfc3-a67cf4512a6f.png)

Reply:
I ran `yarn build` and it helped. So this might help.

Reply:
Changed

Reply:
Would be great if others could check whether it is working now for them @veekeys @FlyersPh9 @gretanausedaite

Reply:
I ran `yarn` and `yarn dev`. The html files seem to work fine